### PR TITLE
Adding config option to immediately edit dimensions

### PR DESCRIPTION
**Pull request by vespakoen**
Here it is again, this time with config option.

Let me know if the default should be on or off, and if the variables / place in the preferences pane are fine with you.

The option does now also work on the angle constraint.

### CODE REVIEW

File: src/confscreen.cpp [hunk 1]
```diff
@@ -342,6 +347,9 @@ void TextWindow::ShowConfiguration() {
         SS.automaticLineConstraints ? CHECK_TRUE : CHECK_FALSE);
     Printf(false, "  %Fd%f%Ll%s  use turntable mouse navigation%E", &ScreenChangeTurntableNav,
         SS.turntableNav ? CHECK_TRUE : CHECK_FALSE);
+    Printf(false, "  %Fd%f%Ll%s  immediately edit dimensions%E",
```
Comment (whitequark) [inline src/confscreen.cpp]: What about `edit newly added dimensions`? The variable naming is fine, but the preference is a bit confusing.

**Comment by whitequark** [comment]
> Let me know if the default should be on or off

The existing behavior should not change for people who are already using SolveSpace.

**Comment by vespakoen** [comment]
Changed the default to false and used your suggested label.
Just have to figure out how to squash these commits together now =)

Will take a bit...

**Comment by whitequark** [comment]
I can just squash while merging in a simple case like this.

**Comment by vespakoen** [comment]
Cool! then I leave it =)
Was just running
```
git reset --soft HEAD~2
```

Which seemed to work allright =)

**Comment by whitequark** [comment]
Usually people use `git rebase -i` which is a bit more flexible, but that works too.

**Comment by vespakoen** [comment]
Indeed, I am not very familiar with git I notices since I mostly only commit and push things with it, should read the manual again...

**Comment by vespakoen** [comment]
Force pushed it anyway ;)

**Comment by whitequark** [comment]
This needs a changelog addition as well.

**Pull request merged**